go/tools/gopackagesdriver/aspect.bzl

```diff
@@ -0,0 +1,116 @@
+load(
```

Comment:
Add copyright boilerplate, here and in other files.

It needs to go pretty much everywhere except build files and test data.